docs/adr/0010-stop-mocking-fetch-in-tests.md

```diff
@@ -0,0 +1,36 @@
+# Replace mocking fetch with `msw`
+
+Date: 2021-01-14
+
+## Status
+
+2021-01-18 Proposed
+
+## Context
+
+For mocking unittests we are using at this moment the `jest-fetch-mock` package. This works but there are new develoments on the market. As the creator of the [Testing Library](https://testing-library.com/) suggests in this [article](https://kentcdodds.com/blog/stop-mocking-fetch), there is a new strategy available that would simplify testing of components that are making fetch calls.
+
+## Technical description
+The `msw` server intercepts the fetch calls and returns the configured response. This is done outside the test file and only overrides of the standard responses from the default configuration need to be added to the respective test file. [Here is the documentation](https://github.com/mswjs/msw).
+
+## Consideratons
+Here are few advantages of introducing this library:
+- All the fetch/mocking can moved to one place (internals/testing)
+- The tests involving fetch will be simpler and easier to read/write
+- Enforces using the correct data/types, no half mock objects
+
```

Comment:
Plus, the e2e tests can also benefit from the msw setup and can use fixtures, the same way the unit tests can.